homeassistant/components/toon/climate.py

```diff
@@ -1,90 +1,164 @@
-"""Support for Toon van Eneco Thermostats."""
+"""Support for Toon thermostat."""
+
+from datetime import timedelta
+import logging
+from typing import Any, Dict, List
+
 from homeassistant.components.climate import ClimateDevice
 from homeassistant.components.climate.const import (
-    STATE_COOL, STATE_ECO, STATE_HEAT, STATE_AUTO,
-    SUPPORT_OPERATION_MODE, SUPPORT_TARGET_TEMPERATURE)
-import homeassistant.components.toon as toon_main
+    STATE_AUTO, STATE_COOL, STATE_ECO, STATE_HEAT, SUPPORT_OPERATION_MODE,
+    SUPPORT_TARGET_TEMPERATURE)
+from homeassistant.config_entries import ConfigEntry
 from homeassistant.const import ATTR_TEMPERATURE, TEMP_CELSIUS
+from homeassistant.helpers.typing import HomeAssistantType
+
+from . import ToonEntity
+from .const import DATA_TOON_CLIENT, DEFAULT_MAX_TEMP, DEFAULT_MIN_TEMP, DOMAIN
+
+DEPENDENCIES = ['toon']
+
+_LOGGER = logging.getLogger(__name__)
 
 SUPPORT_FLAGS = SUPPORT_TARGET_TEMPERATURE | SUPPORT_OPERATION_MODE
 
+MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=5)
+SCAN_INTERVAL = timedelta(minutes=30)
```

Comment:
did you do this on purpose?
```suggestion
SCAN_INTERVAL = timedelta(seconds=30)
```

Reply:
30 seconds will for sure get us banned, so 30 minutes is good.

Reply:
Well, that is actually an error from my end. It should have been seconds indeed.
The previous component used 30 seconds, the custom component uses 30 seconds and the library caches for 30 seconds as well (and has never been an issue).

Reply:
This has been addressed. Thanks again @scheric!